l10n_co_res_partner/README.md

```diff
@@ -0,0 +1,60 @@
+Localización Colombia: Terceros
+======
+
+![main_screenshot.png](http://www.plastinorte.com/images/main_screenshot.png)
+
+What you get
+-----------------
+- Redesign of the contact form due to some local rules that have to apply
```

Comment:
It should be nice extend the statement "Locel Rules" in order to have it here as localization for future references.


Which rules, why, when and what is beyn changed.